lib/game_collection.rb

```diff
@@ -0,0 +1,11 @@
+class GameCollection
+  attr_reader :game_path
+
+  def initialize(game_path)
+    @game_path = game_path
+  end
+
+  def parse
+    CSV.parse("CSV,data,String", **options) do |row|
+  end
+end
```

Comment:
This game_collection file looks different than the one we already have on the game-collection branch. Will that cause problems?